tests/Behat/features/multiple-wysiwyg-configs.feature

```diff
@@ -29,7 +29,7 @@ Feature: multiple elemental blocks with different HTMLEditorConfig instances
     When I click on block 2
     # In CI, the mouse position just happens to produce a tooltip that stops us clicking on the insert link button
     # so we have to move the mouse somewhere else to avoid that
-    And I click on the "input[type='text']" element
+    And I click on the ".element-editor-editform input[type='text']" element
```

Comment:
Needed to avoid a "element not interactable" bug:
https://github.com/creative-commoners/recipe-kitchen-sink/actions/runs/12268277903/job/34279332508